appinventor/appengine/src/com/google/appinventor/client/youngandroid/YoungAndroidFormUpgrader.java

```diff
@@ -1368,6 +1368,10 @@ private static int upgradeNotifierProperties(Map<String, JSONValue> componentPro
       // Added TextInputCanceled & ChoosingCanceled event
       srcCompVersion = 5;
     }
+    if (srcCompVersion < 6) {
```

Comment:
note indentation here is wrong. The project uses 2 spaces for both java and javascript.

Reply:
oops I'll fix that quickly